apollo-router/src/warp_http_server_factory.rs

```diff
@@ -835,7 +835,14 @@ mod tests {
         use tokio::io::{AsyncBufReadExt, AsyncWriteExt, BufReader, Interest};
         use tokio::net::UnixStream;
 
-        let mut stream = UnixStream::connect(addr.to_string()).await.unwrap();
+        let protocol_qualified_name = addr.to_string();
+        let mut stream = UnixStream::connect(
+            protocol_qualified_name
+                .strip_prefix("file://")
```

Comment:
why remove the prefix here?

Reply:
Because the `UnixStream::connect()` call doesn't expect a protocol component in the name. That's one of the reasons why I'm not 100% convinced that we should have protocol in the display name. I just wanted to make file treatment consistent with net.